cli_test.go

```diff
@@ -3,91 +3,100 @@ package main
 import (
 	"os"
 	"reflect"
+	"strings"
 	"testing"
 
+	"github.com/coveo/gotemplate/v3/collections"
 	"github.com/stretchr/testify/assert"
 )
 
+func NewTestApplication(args []string) *TGFApplication {
+	for _, env := range os.Environ() {
+		if strings.HasPrefix(env, "TGF_") {
+			name, _ := collections.Split2(env, "=")
+			os.Setenv(name, "")
+		}
+	}
+	return NewTGFApplication(args)
+}
+
 func TestNewApplicationWithOptionsAndAliases(t *testing.T) {
 	t.Parallel()
 
-	config := &TGFConfig{
-		Aliases: map[string]string{
-			"my_alias":           "--ri --li --stuff3",
-			"my_recursive_alias": "my_alias --with-docker-mount",
-		},
-	}
-
 	tests := []struct {
 		name          string
-		config        *TGFConfig
 		args          []string
 		wantOptions   map[string]interface{}
 		wantUnmanaged []string
 	}{
 		{
-			"Empty", config,
+			"Empty",
 			[]string{},
 			map[string]interface{}{},
 			nil,
 		},
 		{
 			"Managed arg",
-			config, []string{"--ri"},
-			map[string]interface{}{"Refresh": true},
+			[]string{"--ri"},
+			map[string]interface{}{"Refresh": true, "DockerInteractive": true},
 			nil,
 		},
 		{
 			"Managed and unmanaged arg",
-			config, []string{"--li", "--stuff"},
-			map[string]interface{}{"UseLocalImage": true},
+			[]string{"--li", "--stuff"},
+			map[string]interface{}{"UseLocalImage": true, "DockerInteractive": true},
 			[]string{"--stuff"},
 		},
 		{
 			"Alias with an unmanaged arg",
-			config, []string{"my_recursive_alias", "--stuff4"},
-			map[string]interface{}{"Refresh": true, "UseLocalImage": true, "WithDockerMount": true},
```

Comment:
The `WithDockerMount` is not verified anywhere now

Reply:
It was unintentional. Restored!